Sources/arm/Config.hx

```diff
@@ -33,6 +33,15 @@ class Config {
 			Data.getBlob((Path.isProtected() ? Krom.savePath() : "") + "config.arm", function(blob: kha.Blob) {
 				configLoaded = true;
 				raw = Json.parse(blob.toString());
+
+				// Update existing configs to use newly added configurations
+				if (raw.camera_pan_speed == null)
+					raw.camera_pan_speed = 1.0;
+				if (raw.camera_zoom_speed == null)
+					raw.camera_zoom_speed = 1.0;
+				if (raw.camera_rotation_speed == null)
+					raw.camera_rotation_speed = 1.0;
+
```

Comment:
After git pull the config should get auto-reset (https://github.com/armory3d/armorbase/blob/main/Sources/arm/Config.hx#L119) so there is no need for this yet. Will put effort into preserving the config after v1.0.

Reply:
Totally forgot about that. Thank you for the reminder. Will remove the ugly piece of code.